zivid_camera/src/zivid_camera.cpp

```diff
@@ -420,11 +462,25 @@ void ZividCamera::publishFrame(Zivid::Frame&& frame)
   }
 }
 
+bool ZividCamera::shouldPublishColorImg() const
+{
+  return color_image_publisher_.getNumSubscribers() > 0 || use_latched_publisher_for_color_image_;
+}
+
+std_msgs::Header ZividCamera::makeHeader()
+{
+  std_msgs::Header header;
+  header.seq = header_seq_++;
```

Comment:
I think the sequence number has been deprecated for quite a while, but I its sensible to set it like this anyway.